.changeset/yellow-countries-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": minor
+---
+
+feat(Text): add `textAlign` prop
```

Comment:
Why is this added here? Some merge issues?

Reply:
You might be checking on commit https://github.com/razorpay/blade/pull/1016#discussion_r1135396318

Reply:
i think doing "view changes since last" shows this 🤦